### ISSUE TITLE
Doesn't connect to network after reboot

### ISSUE THREAD

**Issue by arkur-code** [RinCat/RTL88x2BU-Linux-Driver#40]
Hello. I have ubuntu 20.04  and after reboot my TP-link t4u not connecting to the network with authentication error. I have to delete the network settings (forget the network) and reconnect with the password.

**Comment by RinCat** [comment]
Hi, the reconnect success?

**Comment by arkur-code** [comment]
Yes. After deleting the wi-fi network settings, the reconnection goes well. I have to delete the settings every time I reboot
